pyro/primitives.py

```diff
@@ -112,7 +112,7 @@ def __init__(self, size, subsample_size, use_cuda=None):
         """
         self.size = size
         self.subsample_size = subsample_size
-        self.use_cuda = torch.Tensor.is_cuda if use_cuda is None else use_cuda
+        self.use_cuda = torch.Tensor().is_cuda if use_cuda is None else use_cuda
```

Comment:
This doesn't seem to be a static method anymore, and does not return a boolean unless instantiated.